codec.go

```diff
@@ -56,7 +59,10 @@ func (c *compressor) write(data []byte) (n int, err error) {
 		n, err = c.z.Write(data)
 		if err == nil {
 			c.z.Flush()
-			// Remove the '00 00 FF FF' trailer from the deflated block
+			// Remove the '00 00 FF FF' trailer from the deflated block:
```

Comment:
Instead of leaving the commented panic code in, could this log a warning?

Reply:
It's just a sanity check, and I don't want it taking up time. If there were an `#if DEBUG` I'd wrap it in that...

Reply:
Ok yeah I can't see a good way to do this without paying a perf penalty, so I guess leaving the commented code in is a good hint for someone who needs to debug this later.